ios/Classes/AblyStreamsChannel.h

```diff
@@ -0,0 +1,26 @@
+//  Copyright (c) 2018 Loup Inc.
```

Comment:
Why do we have Loup's copyright at the top of here when we have renamed it to have an `Ably` prefix? If anything I would expect an attribution or acknowledgement, or that we should be using this in original form. What's going on here? WDYT, @paddybyers ?

Reply:
We should adopt the following practice (shamelessly stolen from Google):

> Files that copy lines of code from third-party files, or that directly translated or derived from third-party files, must retain any copyright notices present in the third-party file(s). AnAbly copyright notice can be added to the license header as a separate copyright notice, beneath any existing notices, but above the license text.
> 
> Any files that are exact copies of files taken from a third-party source must retain any existing copyright notices and must not include any new copyright notices.
> 
> Entirely new, Ably-authored files being added to a project that is forked or derived from a third-party source should use the license header used by other files in the project, omitting any third-party copyright notices. AnAbly copyright notice can be added to the header above the license text.

Reply:
@tiholic can you come up with something ^ or do you need us to assist?

Reply:
I need help, please

Reply:
I'll push a commit.

Reply:
Conformed: https://github.com/ably/ably-flutter/pull/18/commits/86df27e38e1dafead6485b55270e8237e8db8b14

@tiholic, please resolve this conversation if you're happy with my commit.

Reply:
Cool, thanks 😄